test/crd.go

```diff
@@ -22,23 +22,10 @@ import (
 	"strings"
```

Comment:
The file name and the above comment no longer seems to be accurate. This seems to be more of just a prefix library now.. Is there a better spot to put this stuff, or can we at least rename the file from crd.go?

Reply:
Yeah, let's rename in a follow-up.  The cleanup opportunities in `test/` are seemingly endless :)